test/providers/config_test.go

```diff
@@ -0,0 +1,86 @@
+package providers
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/amp-labs/connectors/providers"
+)
+
+func TestReadConfig(t *testing.T) {
+	// Define test cases
+	testCases := []struct {
```

Comment:
Can you add a field to this struct called `description` so each test case has a quick description and if any of them fail, the logged error also contains the description?